_integration-schemas/trello/actions.md

```diff
@@ -0,0 +1,319 @@
+---
+tap: "trello"
+version: "1"
+key: ""
+
+name: "actions"
+doc-link: "https://developer.atlassian.com/cloud/trello/rest/#api-boards-boardId-actions-get"
+singer-schema: "https://github.com/singer-io/tap-trello/blob/master/tap_trello/schemas/actions.json"
+description: |
+  The {{ table.name }} table contains information about the actions within a board in your {{ integration.display_name }} acocunt.
```

Comment:
Looks like this is a child stream of `boards` - does `boards` have to be selected in order for this table to replicate?